control-base/src/main/java/fi/nls/oskari/control/data/MapLayerGroupsHandler.java

```diff
@@ -121,11 +134,39 @@ private void checkForAdminPermission(ActionParameters params) throws ActionExcep
         }
     }
 
-    private void populateFromRequest(ActionParameters params, MaplayerGroup theme) throws ActionException {
-        final Map<String, String> parsed = RequestHelper.parsePrefixedParamsMap(params.getRequest(), PARAM_NAME_PREFIX);
-        for(Map.Entry<String, String> entry : parsed.entrySet()) {
-            // TODO: check against supported locales?
-            theme.setName(entry.getKey(), entry.getValue());
+    protected JSONObject getMapLayerGroup(HttpServletRequest req) throws ActionException {
```

Comment:
This is the same function as `LayerAndGroupOrderHandler#getOrderJSON(HttpServletRequest)` only exception is that the name of the function is different. We should add a `public static JSONObject readJsonFromRequestBody(ActionParameters params);` to `RequestHelper`.